deps/gtest/src/gtest.cc

```diff
@@ -438,6 +446,17 @@ std::string UnitTestOptions::GetAbsolutePathToOutputFile() {
   return result.string();
 }
 
+// Functions for processing the gtest_result_format flag.
```

Comment:
I kept it that way for consistency in case more are added in the future. I see it as more of a list heading than a comment for the one function.